addon/components/inputs/datetime.js

```diff
@@ -1,24 +1,55 @@
 import AbstractInput from './abstract-input'
 import layout from 'ember-frost-bunsen/templates/components/frost-bunsen-input-datetime'
+import Ember from 'ember'
+import { Format, setTime, validateTime } from 'ember-frost-date-picker'
+import moment from 'moment'
 
 export default AbstractInput.extend({
-  // == Component Properties ===================================================
 
-  classNameBindings: ['value:frost-bunsen-has-value'],
+    // == Component Properties ===================================================
 
-  classNames: [
-    'frost-bunsen-input-datetime',
-    'frost-field'
-  ],
+    classNameBindings: ['value:frost-bunsen-has-value'],
 
-  layout,
+    classNames: [
+        'frost-bunsen-input-datetime',
+        'frost-field'
+    ],
 
-  // == Computed Properties ====================================================
+    layout,
 
-  // == Functions ==============================================================
+    dateTimeValue: moment().subtract(1, 'day').subtract(1, 'hour').format(Format.dateTime),
```

Comment:
You really need to have a computed property to respect changes to the value outside of this renderer. Bunsen is designed to let consumers programmatically update the form value if they want and in order to do so when a consumer updates the value the renderer needs to take into account this new value.